src/components/Helpjuice/Helpjuice.test.tsx

```diff
@@ -21,6 +21,17 @@ describe('Helpjuice', () => {
     expect(document.querySelector('path.close')).toBeInTheDocument();
   });
 
+  it('turns header into a link', async () => {
+    render(<Helpjuice />);
+
+    document
+      .getElementsByClassName('hj-swifty')[0]
+      .setAttribute('data-current-question-id', 'article-1');
+    expect(
+      await screen.findByRole('link', { name: 'Article Name' }),
+    ).toHaveAttribute('href', 'https://domain.helpjuice.com/article-1');
+  });
```

Comment:
Can you add another test which changes the currentQuestionId and the test ensures the link is updated?